content/operate/oss_and_stack/stack-with-enterprise/release-notes/redisstack/redisstack-6.2.6-release-notes.md

```diff
@@ -0,0 +1,1278 @@
+---
+Title: Redis Stack 6.2.6 release notes
+alwaysopen: false
+categories:
+- docs
+- operate
+- stack
+description: Redis Stack 6.2.6 release notes.
+linkTitle: v6.2.6-v0 (December 2021)
```

Comment:
Please verify the date. This is what I saw on the repo site: 2022-11-23. So maybe this should be November 2022 instead of December 2022? In either case, December 2021 is clearly not correct.

```suggestion
linkTitle: v6.2.6-v0 (December 2022)
```

Reply:
thank you